app/cdap/components/WranglerGrid/NestedMenu/MenuComponent/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import MenuItemComponent, { IMenuItem } from 'components/WranglerGrid/NestedMenu/MenuItemComponent';
+import { MenuComponent } from 'components/common/MenuContainer';
```

Comment:
`MenuComponent` contains `MenuComponent`? That's confusing

Reply:
MenuComponent is Styled Component named the same as the Component in which it is used. So for more clarity renamed the styled MenuComponent as CustomizedMenuComponent.
import { CustomizedMenuComponent } from 'components/common/MenuContainer';